docs/www/nodule-grid.js

```diff
@@ -377,6 +377,368 @@
+                        "Finding": "Pulmonary nodule",
+                        "Attenuation Characteristic": "Unknown",
+                        "Radiographic Lesion Margin": "Lesion with circumscribed margin",
+                        "Finding site": "Lower lobe of right lung",
+                        "Lung-RADS assessment": "Lung-rads 0",
+                        "Lesion Review Status": "Accepted",
+                        "Maximum 2D diameter": "5.5",
+                        "Maximum 3D diameter": "6.2",
+                        "Maximum perpendicular 2D diameter": "4.2",
+                        "Mean 2D diameter": "4.9",
+                        "Volume": "34.5"
+                    },
+                    {
+                        "Tracking Identifier": "R9",
+                        "Tracking Unique Identifier": "1.3.12.2.1107.5.8.21.192168059.30000021092410281563200001347",
+                        "Finding": "Pulmonary nodule",
+                        "Attenuation Characteristic": "Solid",
+                        "Radiographic Lesion Margin": "Lesion with circumscribed margin",
+                        "Finding site": "Upper lobe of right lung",
+                        "Lung-RADS assessment": "Lung-rads 2",
+                        "Lesion Review Status": "Accepted",
+                        "Maximum 2D diameter": "5.4",
+                        "Maximum 3D diameter": "5.4",
+                        "Maximum perpendicular 2D diameter": "4.0",
+                        "Mean 2D diameter": "4.7",
+                        "Volume": "21.4"
+                    }
+                ];
+
+                //VAPALS.displayNotification("Updating Data");
+                if (!confirm("Do you really want to do the data update?")) {
+                    return;
+                }
+
+                // if number of nodules shown is less than the number of entries in theData array then increase the number of nodules to they match
+                while (settings.getNoduleCount() < theData.length) {
+                    _addNodule()
+                }
+
+                noduleId = 1;
+                theData.forEach(obj => {
+                    Object.entries(obj).forEach(([key, value]) => {
+                        // Need to work: “Finding”: “Pulmonary nodule”,
+
+                        // Need to work: “Finding site”: “Upper lobe of right lung”
+                        //So assuming using the "Most likely location" field ( cect1ll ) of the form.
+                        //“Upper lobe of right lung” = "RUL"
+                        //"Upper lobe of left lung" = "LUL"
+                        //"Lower lobe of right lung" = "RLL"
+                        //"Lower lobe of left lung" = "LLL"
+                        if (key === "Finding site") {
+                            ll = "#cect" + noduleId + "ll";
+                            if (!$(ll).hasClass("update-data")) {
```

Comment:
Do not continuously call $(ll) here. You can just set a single variable equal to the jquery object. e.g. const $field = $(fieldSelector)